lib/index_documents.rb

```diff
@@ -0,0 +1,65 @@
+require_relative "../app"
+require 'gds_api/publishing_api_v2'
+
+class IndexDocuments
+  def process(message)
+    links = message.payload["links"].try(:dup)
+
+    if links
+      links.each do |(k, vs)|
```

Comment:
We don't want to use all the link types - only the ones which come from content-tagger.  Also, the values for link types don't always match the field names in rummager (eg, the link type of `topics` maps to the `specialist_sectors` field in rummager), so I think we'll need some custom logic for each link type.